app/presenters/content_item_presenter.rb

```diff
@@ -37,7 +37,7 @@ def present_linked_item(linked_item)
   end
 
   def api_url(item)
-    @api_url_method.call(item.base_path)
+    @api_url_method.call(item.base_path.sub(%r{^/}, ""))
```

Comment:
the need to strip the leading slash is coupled to the implementations of the `content_item_api_url` and `content_item_url` methods which are a few hops away in the code. 

How about refactoring this so that `@api_url_method.call` accepts an `item` and does what it needs. Or even continues to accept `item.base_path` and does the stripping itself. That moves the knowledge about the url making into one place.